src/backend/app.js

```diff
@@ -21,10 +22,80 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+// Route to respond with all meals in the future
+router.get("/future-meals", async (req, res) => {
+  try {
+    const futureMeals = await knex.raw(
+      "SELECT * FROM meals WHERE `when` > NOW()"
+    );
+    res.json(futureMeals[0]);
```

Comment:
You can sent it with a status code to make sure the user knows that the expected data is returned